### PR TITLE
Fixes #3570 & #3179 merged reading_text_size_fragment into single file and migrated app b…

### LINKED ISSUE #3570: Merge reading_text_size_fragment.xml into single xml file
Currently there are 2 versions of `reading_text_size_fragment.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by mukulasija**
…ar layout to reading_text_size_activity

Fixes #3570 & #3179 
moved the reading_list_app_bar_layout from reading_text_size_fragment to reading_text_size_activity 
and merged 2 reading_text_size_fragment files into single file

op

tnp

ol

tnl

ol

ool

op

oop

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by mukulasija** [comment]
@ARJUPTA PTAL

**Review by Arjupta — COMMENTED** [review]
@mukulasija please take a look at the failing checks

**Comment by mukulasija** [comment]
resolved the issues with failing lint checks.. @ARJUPTA PTAL

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 1]
```diff
@@ -399,4 +399,7 @@
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
 
+  <!--  ReadingTextSizeFragment-->
+  <dimen name="reading_text_size_fragment_padding_bottom">132dp</dimen>
+
 </resources>
```
Comment (Arjupta) [inline app/src/main/res/values-sw600dp-land/dimens.xml:412]: As `reading_text_size_fragment_padding_bottom` has the same value as in `dimens.xml` so you can delete extra declaration from here.

File: app/src/main/res/values-sw600dp-port/dimens.xml [hunk 2]
```diff
@@ -401,4 +401,7 @@
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
 
+  <!--  ReadingTextSizeFragment-->
+  <dimen name="reading_text_size_fragment_padding_bottom">132dp</dimen>
+
 </resources>
```
Comment (Arjupta) [inline app/src/main/res/values-sw600dp-port/dimens.xml:414]: Ditto

File: app/src/main/res/layout/reading_text_size_fragment.xml [hunk 3]
```diff
@@ -9,42 +9,6 @@
       type="org.oppia.android.app.options.ReadingTextSizeSelectionViewModel" />
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
-
```
Comment (Arjupta) [inline app/src/main/res/layout/reading_text_size_fragment.xml:15]: I am not sure whether we have to remove both Constraint and Frame Layout from here, though we know only keeping a RecylerView makes a lot of sense. @rt4914 what are your reviews over this?

File: app/src/main/java/org/oppia/android/app/options/ReadingTextSizeActivityPresenter.kt [hunk 4]
```diff
@@ -26,6 +28,13 @@ class ReadingTextSizeActivityPresenter @Inject constructor(
     this.fontSize = fontSize
   }
 
+  private fun setToolbar() {
+    val readingTextSizeToolbar: Toolbar = activity.findViewById(R.id.reading_text_size_toolbar)
+    readingTextSizeToolbar.setNavigationOnClickListener {
+      activity.onBackPressed()
+    }
+  }
```
Comment (Arjupta) [inline app/src/main/java/org/oppia/android/app/options/ReadingTextSizeActivityPresenter.kt:36]: You forgot to include the selectedReadingTextSize in the result for this Activity, compare it with the fragment to understand the difference

**Review by Arjupta — CHANGES_REQUESTED** [review]
@mukulasija PTAL

File: app/src/main/java/org/oppia/android/app/options/ReadingTextSizeActivityPresenter.kt (hunk 4 above)
Comment (mukulasija) [reply app/src/main/java/org/oppia/android/app/options/ReadingTextSizeActivityPresenter.kt:36]: @ARJUPTA  what I thought was, that functions onBackPressed is already overridden in ReadingTextSizeActivity.kt file which will handle everything.

Comment (Arjupta) [reply app/src/main/java/org/oppia/android/app/options/ReadingTextSizeActivityPresenter.kt:36]: `ReadingTextSizeActivity` will be able to handle backPress but we also need to set the updated test size data in the activity result (which can be read by previous activity).

File: app/src/main/res/layout/reading_text_size_fragment.xml (hunk 3 above)
Comment (rt4914) [reply app/src/main/res/layout/reading_text_size_fragment.xml:15]: The FrameLayout was used to show the shadow correctly, so if the final output does show shadow correctly we can remove it.

**Comment by yash10019coder** [comment]
@mukulasija you have to add Fixes #3179 in new line for every new issue we write seperate fixes as if you see in the issue your PR isn't linked PTAL

**Pull request closed without merging**